### ISSUE TITLE
Breaks under mac gui tools

### ISSUE THREAD

**Issue by s10wen** [leakypixel/captain-githook#1]
When trying to commit, I get the following errors:

GitHub Desktop

```
env: node: No such file or directory
 (1)
```

SourceTree:

```
git -c diff.mnemonicprefix=false -c core.quotepath=false -c credential.helper=sourcetree commit -q -F /var/folders/gt/_15xqls50s91rh11j8zm4n99z984qp/T/SourceTreeTemp.2svl5p 
env: node: No such file or directory
Completed with errors, see above
```

**Comment by leakypixel** [comment]
After investigation, it appears that Mac OSX GUI tools do not have the full path variable that's available to command-line tools. 

Working on a fix now.

**Comment by leakypixel** [comment]
Fixed in [v1.0.3](https://github.com/leakypixel/captain-githook/releases/tag/v1.0.3).
